Burrows-Wheeler/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿const int alphabetSize = 256;
+
+(string, int) Transform(string originalString)
+{
+    if (originalString == "")
+    {
+        return ("", 0);
+    }
+
+    int[] suffixArray = Enumerable.Range(0, originalString.Length).ToArray();
+
+    Array.Sort(suffixArray, (i, j) => string.Compare($"{originalString.Substring(i)}{originalString.Remove(i)}", $"{originalString.Substring(j)}{originalString.Remove(j)}"));
+
+    char[] resultArray = new char[originalString.Length];
```

Comment:
```suggestion
    var resultArray = new char[originalString.Length];
```
не пишите тип дважды